.github/workflows/build.yaml

```
@@ -27,45 +27,46 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: ["8.3"]
-                symfony: ["^5.4.21", "^6.4"]
-                sylius: ["~1.13.0", "~1.14.0"]
-                database: ["mysql", "postgres"]
-                mysql: ["8.4"]
-                postgres: ["15.8"]
+                php: ["8.2", "8.3"]
+                symfony: ["^6.4", "^7.1"]
+                sylius: ["~2.0"]
```

I would be in favour of limiting it to just one minor, as we did before
```suggestion
                sylius: ["~2.0.0"]
```